### ISSUE TITLE
Publish to Sixteen Colors

### ISSUE THREAD

**Issue by lordscarlet** [sixteencolors/js-textmode-editor#29]

**Comment by ispyhumanfly** [comment]
This is an absolute must in my opinion. And should be required for when we actually release this officially as draw.sixteencolors.net. I freaking love this thing! And the first thing I wanted to do when I was finished drawing a simple Ansi was publish to 16c (instinctively).

**Comment by lordscarlet** [comment]
Yeah, we're skirting this issue right now by having draw.sixteencolors.net not go DIRECTLY to the editor, but we need some sort of interacation between them. However, paste.sixteencolors.net shows us quickly that it can be dangerous -- people can put up garbage very easily.
